src/components/FormField.js

```diff
@@ -0,0 +1,121 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+class FormField extends React.Component {
+  render () {
+    return (
+      <div
+        className='mx-form-field-component'
+        style={[
+          styles.component, this.props.style,
+          { height: this.props.height + 'px', width: this.props.width + 'px' }
```

Comment:
Not sure if we always have to append 'px' but it defaults to pixels.
`{ height: this.props.height, width: this.props.width }`